app/workers/education_form/templates/5495.erb

```diff
@@ -67,16 +65,6 @@ SSN:   <%= @applicant.veteranSocialSecurityNumber %>
 
 VA File Number: <%= @applicant.vaFileNumber %>
 
-Branch of Service:   <%= @applicant.serviceBranch %>
```

Comment:
it's not supposed to be the applicant's own service, that info is in the in the toursOfDuty array